__tests__/models/output.test.js

```diff
@@ -97,4 +98,24 @@ test('Script', () => {
   expect(() => {
     parseP2PKH(o1.script.slice(10), network);
   }).toThrowError(ParseScriptError);
+});
+
+test('Script data', () => {
```

Comment:
Shouldn't this test be included in a new file `__tests__/models/script_data.test.js`?

If I understand the naming pattern correctly, each model should have its own test file, right?